addons/website_event_sale/models/product.py

```diff
@@ -14,10 +14,3 @@ def _is_add_to_cart_allowed(self):
         self.ensure_one()
         res = super()._is_add_to_cart_allowed()
         return res or any(event.website_published for event in self.event_ticket_ids.event_id)
-
-class ProductTemplate(models.Model):
-    _inherit = 'product.template'
-
-    @api.model
-    def _get_product_types_allow_zero_price(self):
-        return super()._get_product_types_allow_zero_price() + ["event"]
```

Comment:
same as below

Reply:
FYI
This method does called now for events so removing this method for event, still this case happen for booth and course I guess
https://github.com/odoo/odoo/pull/158754

Need to check with SM PO if we can do same for booth and course 🤔